### DISCUSSION TITLE
Latest pybricks version running on EV3

### DISCUSSION THREAD

**Discussion by MichaelHausherr** [pybricks/support#571] [Q&A] [answered]
Hi everyone. First of all, Happy New Year and a big thank you for the great work behind pybricks.
While our children are making their first steps programming the EV3 with micropython and pybricks (they have used the labview based software before) I'm trying to understand the relation between the different pybricks versions and the various bricks/hubs supported.
Is every pybricks version only supporting a specific group of bricks/hubs? Or is it more like this: newer pybricks versions add features and support for more bricks and hubs? Would this mean that we could also use the latest V3.1 with the EV3? Would this make sense and how could this be done? Thanks in advance.

**Comment by dlech** [comment, marked as answer]
The 3.x series has been focused on adding support for new hubs. I would suggest sticking with the 2.x version that ships with ev3dev unless there is some specific issue that you run into that is solved in the 3.x version. (You can download the `pybricks-micropython` executable from [successful CI builds](https://github.com/pybricks/pybricks-micropython/actions/workflows/build.yml?query=is%3Asuccess) and install it in `/usr/local/bin/` on the EV3 if you really want to try a newer version - but we don't regularly test it on EV3, so we usually only suggest this for "pro" users).
